workers/unity/Packages/io.improbable.gdk.core/ReactiveComponents/Authority/AuthorityChanges.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !DISABLE_REACTIVE_COMPONENTS
+﻿#if EUSE_LEGACY_REACTIVE_COMPONENTS
```

Comment:
Typo

Reply:
Yeah I hit this in the test-project. Fixed in "double woops"